docs/rules/0132/request-required-fields.md

```diff
@@ -1,86 +0,0 @@
----
```

Comment:
I'm not sure this omission was intentional from the aeps - maybe something we should bring up?

Reply:
maybe @rofrankel might want to weigh in here?

Reply:
Yep, put this rule back in.